eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     # Linux arm64
     - ${{ if eq(parameters.platform, 'linux_arm64') }}:
-      - (Ubuntu.2204.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8
+      - (Ubuntu.2410.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.10-helix-arm64v8
```

Comment:
Could someone enlighten me about the naming scheme? I am bit confused about Ubuntu 24.10 still having `Ubuntu.2204.ArmArch.Open` in the name.

Reply:
This is Ubuntu 24.10 container running on Ubuntu 22.04 physical machine (ie the kernel is from Ubuntu 22.04).

Reply:
Thanks!

Reply:
this comes from "fixes" we put in for https://github.com/dotnet/runtime/issues/67353 @filipnavara. 
Do you want to disable the test as part of this PR @richlander? 
NTLM was broken on Ubuntu for a while and it seems like it is finally fixed.

Reply:
I don't have context to make that decision. Can you elaborate?

Reply:
@tarekgh already pushed the change @richlander. No need to worry. We will make proper change later after more testing on new images.

Reply:
Sounds good. I will be bringing in Debian 13, shortly. 24.10 will be replaced by 25.04 before long. My intent is to keep us closer to tip.